scripts/appleseed.package.py

```diff
@@ -507,7 +507,7 @@ def add_python_to_stage(self):
         shutil.copy(os.path.join(self.settings.python_path, "README.txt"), "appleseed/python27")
 
     def copy_qt_framework(self, framework_name):
-        src_filepath = os.path.join(self.settings.qt_runtime_path, framework_name + "4" + ".dll")
+        src_filepath = os.path.join(self.settings.qt_runtime_path, framework_name + "5" + ".dll")
         dst_path = os.path.join("appleseed", "bin")
```

Comment:
Did you actually check that this works?  The name of the file under Qt 4 was QtCore4.dll.  Under Qt5 it’s Qt5Core.dll